src/app/data-catalogue/components/service-detail/service-detail.component.ts

```diff
@@ -50,6 +50,7 @@ export class ServiceDetailComponent extends AbstractBaseDetailComponent<ServiceM
       itemTitle: serviceMetadata.name,
       category: 'Geodienst',
       shortDescription: serviceMetadata.description,
+      imageUrl: null,
```

Comment:
Why are these hardcoded to `null`, seeing that the generated interface has either a `LinkObject` _or_ `null`?

Reply:
For the other three types except for map, we receive an image object (sometimes), but the kanton doesn't want us to dispaly them (yet), so we just block the display of the images if the metadata element is not a map